GameServerCore/Domain/GameObjects/IObjAiBase.cs

```diff
@@ -70,6 +70,29 @@ public interface IObjAiBase : IAttackableUnit
         /// Unit this AI will auto attack when it is in auto attack range.
         /// </summary>
         IAttackableUnit TargetUnit { get; set; }
+        /// <summary>
+        /// A unit's acquisition range is the distance at which they will automatically acquire attack targets, referred to as 'auto-attacking'.
+        /// </summary>
+        int AcquisitionRange { get; set; }
```

Comment:
Move this to AIScriptMetadata.

Reply:
Moved to `Stats`, `StatsModifier`, and `CharData`